src/odoc/html_page.ml

```diff
@@ -16,7 +16,17 @@
 
 open StdLabels
 
-let from_odoc ~env ?theme_uri ~output:root_dir input =
+let to_html_tree_page ?theme_uri ~lang v =
+  match lang with
+  | Html.Html_tree.Reason -> Html.To_html_tree.RE.page ?theme_uri v
```

Comment:
I don't like this... I felt like we should use something like first-class-modules, but I was not entirely sure how to do this the right way... feedback welcome!